src/coreclr/nativeaot/Common/src/Internal/Runtime/TransitionBlock.cs

```diff
@@ -218,40 +228,42 @@ struct ArchitectureConstants
     }
 #elif TARGET_ARM
 #pragma warning disable 0169
-    struct ReturnBlock
+#pragma warning disable CA1823
```

Comment:
`[StructLayout(LayoutKind.Sequential)]` also suppresses this warning and makes it clearer that the code depends on the layout so I'd suggest replacing the `#pragma`s with it everywhere.

Reply:
Yeah, I did it this way because the code was already suppressing CS0169 so it seemed more fitting to suppress CA1823 too.